src/main/java/com/minecolonies/coremod/entity/ai/minimal/EntityAICitizenWander.java

```diff
@@ -66,22 +71,23 @@ public boolean shouldExecute()
      *
      * @return True when age => 100, otherwise false.
      */
-    private boolean isToOld()
+    private boolean isTooOld()
     {
-        return citizen.getAge() >= 100;
+        return citizen.getGrowingAge() >= OLD_AGE;
     }
 
     private boolean checkForRandom()
     {
         return citizen.getRNG().nextInt(120) != 0;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 120 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)